src/hash/rescue/rpx/digest.rs

```diff
@@ -423,6 +426,10 @@ impl Serializable for RpxDigest {
     fn write_into<W: ByteWriter>(&self, target: &mut W) {
         target.write_bytes(&self.as_bytes());
     }
+
+    fn get_size_hint(&self) -> usize {
+        self.as_bytes().get_size_hint()
```

Comment:
Question: should this return just `Self::SERIALIZED_SIZE`?

Reply:
Yeah, sounds better actually!